lib/galaxy/managers/workflows.py

```diff
@@ -842,35 +819,7 @@ def __walk_step_dicts( self, data ):
 
             yield step_dict
 
-    def __track_module_from_dict( self, trans, steps, steps_by_external_id, step_dict, exact_tools=False ):
```

Comment:
I dislike this refactoring. I get the function has a crappy name - but the operations should be distinct. I prefer the simpler lower mccabe complexity functions - and the ability to see the relatively stateless __module_from_dict from the larger function that tracks a bunch of state as it goes.

This is a like -0 thing, I'm not going to hold up the PR based on it - just expressing my desire and design aesthetic.